samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -9,4 +9,21 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 # LC3 lib requires floating point support in the c-lib NEWLIB is one way of getting that.
 CONFIG_NEWLIB_LIBC=y
 CONFIG_MAIN_STACK_SIZE=4096
+
+# USB Audio related configs
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"
+CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
+CONFIG_USB_DEVICE_AUDIO=y
+
+# Two streams in one subgroup
+CONFIG_BT_ISO_MAX_CHAN=2
+CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=2
+CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
+
+# Audio buffer
+CONFIG_RING_BUFFER=y
+
+# Additional BT settings
+CONFIG_BT_ECC=y
```

Comment:
What is `CONFIG_BT_ECC` being used for?

Reply:
I saw there was a new dependency for that with the tinycrypt config to be enabled.  Maybe we don't need it anymore for the zephyr controller?

Reply:
`BT_ECC` is enabled by default if we support SMP:

```c
config BT_ECC
	bool "ECDH key generation support"
	default y if BT_SMP && !BT_SMP_OOB_LEGACY_PAIR_ONLY
	help
	  This option adds support for ECDH HCI commands.
```

But I see that for the broadcast source we probably don't enable BT_SMP as we don't have any support for connections, so we may need to enable this specifically indeed. 

@aescolar is also modifying this file in https://github.com/zephyrproject-rtos/zephyr/pull/64704, where we've decided that enabling Tinycrypt by default for all boards doesn't hurt, so the changes in https://github.com/zephyrproject-rtos/zephyr/pull/64704/files#diff-cfb10a5de807c254935f69d67e5af3e7a303b417df1ae3a16623feeb7235bf85R17 will likely conflict with this as well (just FYI).

Reply:
I see you have a TX MTU there at 60 - will that work even if we only need 40 (or does it have to be exact)? - I'll try (is needed for 24KHz, methinks)

Reply:
MTU of 60 is required for 24_2_1.

The MTU value is a maximum, so it does not need to be exact; it just needs to be >= the maximum value we need